pxb-tag/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "@pxblue/tag",
+    "description": "A script that can be used to automatically tag packages in a continuous integration pipeline.",
+    "version": "1.0.0",
+    "author": "Power Xpert Blue <pxblue@eaton.com> (https://github.com/pxblue)",
+    "license": "BSD-3-Clause",
+    "repository": {
+        "url": "https://github.com/pxblue/automation",
+        "type": "git"
+    },
+    "bugs": {
+        "url": "https://github.com/pxblue/automation/issues"
+    },
+    "keywords": [
+        "pxblue",
+        "automation",
+        "tag",
+        "git",
+        "release"
+    ],
+    "main": "index.js",
+    "bin": {
+        "pxb-tag": "scripts/tag.sh",
+        "pxb-parse-changelog": "scripts/parse-changelog.js"
+    },
+    "files": [
+        "scripts/",
+        "README.md",
+        "CHANGELOG.md",
+        "LICENSE",
+        "package.json"
+    ],
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 0",
+        "tag-package": "npx -p @pxblue/tag pxb-tag"
```

Comment:
should this one be using the local copy instead of npx. E.g. if you are tagging the tag package, would you want to be using the local changes that you may have just made? Or does it not matter because this happens after the publish step so any local changes would already be in npm?

Reply:
You're right, npx doesn't matter in this case because the tag step always comes after the publish step.  By the time we run npx, the current build will already be in npm.